shuup/core/models/_product_shops.py

```diff
@@ -32,6 +34,24 @@
 mark_safe_lazy = lazy(mark_safe, six.text_type)
 
 
+class SimpleSupplierInternalProductTypeProvider:
```

Comment:
this has nothing to do with SimpleSupplier. also this should be moved to Product instead of ShopProduct